src/utils/errors.ts

```diff
@@ -36,16 +36,38 @@ const requestQuery = (response: HttpResponse): string => {
 }
 
 export const extractErrorMessageFromEnvelope = (z: ZObject, response: HttpResponse): ErrorDetails => {
-  const errorResponse = parseErrorResponse(z, response.content)
   return {
     statusCode: response.status,
-    statusMessage: extractStatusMessage(errorResponse),
-    errorDetails: extractErrorDetails(errorResponse),
+    ...buildErrorMessageDetails(z, response),
     query: requestQuery(response),
     url: response.request.url
   }
 }
 
+export const buildErrorMessageDetails = (z: ZObject, response: HttpResponse): { statusMessage: string, errorDetails: string[] } => {
```

Comment:
Pretty long line, eslint does not complain about it?

Reply:
Hmm, probably I have misconfigured it, will doubl check configuration